src/azure-cli/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -81,11 +81,13 @@ def load_command_table(self, _):
         g.custom_command('abort', 'revert')
 
 # NetwrokRuleSet Region
-    with self.command_group('servicebus namespace network-rule', sb_namespace_util, client_factory=namespaces_mgmt_client_factory, resource_type=ResourceType.MGMT_SERVICEBUS) as g:
-        g.custom_command('add', 'cli_networkrule_createupdate', validator=validate_subnet)
-        g.command('list', 'get_network_rule_set')
-        g.custom_command('remove', 'cli_networkrule_delete', validator=validate_subnet)
-        g.custom_command('update', 'cli_networkrule_update')
+    with self.command_group('servicebus namespace network-rule', sb_namespace_util, deprecate_info=self.deprecate(redirect='eventhubs namespace network-rule-set'), client_factory=namespaces_mgmt_client_factory, resource_type=ResourceType.MGMT_SERVICEBUS) as g:
```

Comment:
`servicebus namespace network-rule` will be redirected to `eventhubs namespace network-rule-set`??????
`servicebus` -> `eventhubs`???? You sure??????

Reply:
Sorry, My Bad.
Resolved.